bear.py

```
@@ -62,13 +58,13 @@ def reproduct(self, another_bear, the_field):
         aggression = self.create_value_for_feature(self.aggression, another_bear.aggression)
         cowardice = self.create_value_for_feature(self.cowardice, another_bear.cowardice)
 
-        activity_level = random.random()
-        smell = random.random()
-        aggression = random.random()
-        cowardice = random.random()
         life_count = constants.INITIAL_LIFE_LEVEL
         random_empty_cell = the_field.get_random_empty_cell()
 
+        if random_empty_cell is None:
+            print >> out_file, "No place for baby bear"
+            sys.exit()
```

You should throw an exception in this case